website/client/package-lock.json

```diff
@@ -10094,6 +10094,12 @@
           "resolved": "https://registry.npmjs.org/emoji-regex/-/emoji-regex-8.0.0.tgz",
           "integrity": "sha512-MSjYzcWNOA0ewAHpz0MxpYFvwg6yjy1NG3xteoqz644VCo/RPgnr1/GGt+ic3iJTzQ8Eu3TdM14SawnVUmGE6A=="
         },
+        "emojis-list": {
+          "version": "3.0.0",
+          "resolved": "https://registry.npmjs.org/emojis-list/-/emojis-list-3.0.0.tgz",
+          "integrity": "sha512-/kyM18EfinwXZbno9FyUGeFh87KC8HRQBQGildHZbEuRyWFOmv1U10o9BBp8XVZDVNNuQKyIGIu5ZYAAXJ0V2Q==",
+          "optional": true
+        },
```

Comment:
These package-lock updates don't look relevant. Is any of this actually meant to be part of the PR? If not, this file can be removed.

Reply:
I don't really know why those we're added - let me delete it and reinstall it

Reply:
when I delete and let it reinstall it 

it re-adds all of it again - I'll just use the develop version but its still odd that it changes these 


maybe we need to go stricter on the packages for that, e.g.
`   "uuid": "8.3.2",`
instead of
`   "uuid": "^8.3.2",`